va/va.h

```diff
@@ -1582,6 +1582,11 @@ typedef enum {
     /** \brief Surface usage hint, gives the driver a hint of intended usage 
      *  to optimize allocation (e.g. tiling) (int, read/write). */
     VASurfaceAttribUsageHint,
+    /** \brief List of possible DRM format modifiers (pointer, write).
+     *
+     * The value must be a pointer to a VADRMFormatModifierList.
+     */
+    VASurfaceAttribDRMFormatModifiers,
```

Comment:
Perhaps this should also be readable, returning the list of supported modifiers to `vaQuerySurfaceAttributes()`?

(Unclear how the memory works in that case, maybe it needs to be a pointer to something allocated inside the driver.)

Reply:
I've just matched what `VASurfaceAttribMemoryType` does here.

Reply:
Yeah, the memory ownership issue is the reason why I haven't added this as a readable attrib. Since the supported modifiers depend on the VAConfig, the driver can't have a single global modifier list.

Maybe a better path forward would be to return one attrib per supported modifier, but then we can't re-use the same attrib.